.github/workflows/jlab-a11y-test.yml

```diff
@@ -16,63 +33,29 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  jlab-a11y:
+  print-info:
     runs-on: ubuntu-latest
-    defaults:
-      run:
-        working-directory: ./testing/jupyterlab
-        shell: bash -l {0}
     steps:
-      - name: Checkout repository 🛎
-        uses: actions/checkout@v3
-
-      - name: Cache conda 🧠
-        uses: actions/cache@v2
-        env:
-          # Increase this value to reset cache if environment.yml has not changed
-          CACHE_NUMBER: 0
-        with:
-          path: ~/conda_pkgs_dir
-          key: ${{ runner.os }}-conda-${{ env.CACHE_NUMBER }}-${{
-            hashFiles('environment.yml') }}
-
-      - name: Install miniconda 🐍
-        uses: conda-incubator/setup-miniconda@v2
-        with:
-          miniforge-variant: Mambaforge
-          miniforge-version: latest
-          use-mamba: true
-          environment-file: ./testing/jupyterlab/environment.yml
-          activate-environment: a11y-tests
-          use-only-tar-bz2: true
+      - run: echo "🎉 The job was automatically triggered by a ${{ github.event_name }} event."
+      - run: echo "🐧 This job is now running on a ${{ runner.os }} server hosted by GitHub!"
+      - run: echo "🔎 The name of your branch is ${{ github.ref }} and your repository is ${{ github.repository }}."
+      - run: echo "🗂 Workspace is ${{ github.workspace }}"
+      - run: echo "inputs.repository = ${{ inputs.repository }}"
+      - run: echo "inputs.ref = ${{ inputs.ref }}"
+      - run: echo "github.workspace = ${{ github.workspace }}"
+      - run: echo "github.sha = ${{ github.sha }}"
+      - run: echo "github.repository = ${{ github.repository }}"
+      - run: echo "github.server_url = ${{ github.server_url }}"
 
-      - name: Sanity checks on environment 🔍
-        run: |
-          conda info
-          conda env list
-          echo $CONDA_PREFIX
 
-      - name: Install node dependencies 🧶
-        run: |
-          conda run --prefix $CONDA_PREFIX npm install
-          conda run --prefix $CONDA_PREFIX npx playwright install chromium
-
-      - name: Run tests  ✅
-        continue-on-error: true
-        run: |
-          conda run --prefix $CONDA_PREFIX npm test
-
-      - name: Get run refs 🏷
-        id: getrefs
-        run: |
-          export raw_branch=${GITHUB_REF#refs/heads/}
-          echo "::set-output name=branch::${raw_branch//\//-}"
-          echo "::set-output name=sha8::$(echo ${GITHUB_SHA} | cut -c1-8)"
+  jlab-a11y:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
 
-      - name: Upload testing results 📤
-        uses: actions/upload-artifact@v3
-        # using default retention policy = 90 days
+      - name: Run testing action
```

Comment:
If we want to be able to run the testing action from the JupyterLab repo, I need to be put the checkout step above into the action itself, but I'm not quite sure how to express in a workflow: checkout the repo that this action belongs to, **not** the repo that called this action.